rooms/rooms.py

```diff
@@ -0,0 +1,5 @@
+class Rooms(object):
```

Comment:
I think this class would be better of named `Room` instead of `Rooms` to be consistent with the singular format.